assets/js/atomic/blocks/product/add-to-cart/product-types/variable/variation-attributes/attribute-select-control.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { decodeEntities } from '@wordpress/html-entities';
+import { SelectControl } from '@wordpress/components';
+
+// Default option for select boxes.
+const selectAnOption = {
+	value: '',
+	label: __( 'Select an option', 'woo-gutenberg-products-block' ),
```

Comment:
Should this option be [disabled](https://github.com/WordPress/gutenberg/tree/master/packages/components/src/select-control#options) so it can't be manually selected by customers?

Reply:
User should be able to remove selections because other selects are narrowed down by available options.